core/state_transition.go

```diff
@@ -66,7 +65,7 @@ type StateTransition struct {
 }
 
 func (st *StateTransition) fee() *big.Int {
-	return new(big.Int).Add(st.gasPrice, st.minerTip)
+	return new(big.Int).Set(st.gasPrice)
```

Comment:
[nit] probably don't need a new big int here

Reply:
okay, can change since this is only read